kingdom_sdk/domain/exception.py

```diff
@@ -0,0 +1,24 @@
+from kingdom_sdk import config
+
+
+class KingdomError(Exception):
+    _message: str
+    _code: str
+
+    def __init__(self, message: str, code: str) -> None:
+        self._message = message
+        self._code = code
+
+    def __repr__(self) -> str:
+        if config.is_debug_active():
+            return f"{self.__class__.__name__}: {self.message} [{self.code}]"
+        else:
+            return self.code
```

Comment:
I don't think so that's a good idea to expose classes names in error messages in the production environment. My idea is show detailed info to front-end only in sandbox environment, and a short code in production. With the code, we can track what happened in the Syslog, where the full message will be printed. The __repr__ will be used only by the `entrypoint_error_resolver` (fastapi or graphql).